packages/react/src/hooks/useRegister.ts

```diff
@@ -15,6 +15,10 @@ type UseRegisterReturn = HooksReturn<
   }
 >;
 
+const getAccountFromDidPkh = (didPkh: string) => {
```

Comment:
Can we move this to a file like `/utils/accountParser.ts` or some other name and provide more info with JSDoc? What is `didPkh` mean? Why did required to make sure the account string includes three parts?

Reply:
Ah fair, I just carried this over from notify client